cmake/modules/FindNVJPEG.cmake

```diff
@@ -31,11 +32,11 @@ find_package_handle_standard_args(NVJPEG
     REQUIRED_VARS NVJPEG_INCLUDE_DIR NVJPEG_LIBRARY
     VERSION_VAR NVJPEG_VERSION)
 
-message(${CUDA_TOOLKIT_ROOT_DIR})
+# message(${CUDA_TOOLKIT_ROOT_DIR})
 if(NVJPEG_FOUND)
   # set includes and link libs for nvJpeg
-  set(CMAKE_REQUIRED_INCLUDES ${CUDA_INCLUDE_DIRS})
-  set(CMAKE_REQUIRED_LIBRARIES ${NVJPEG_LIBRARY} "-L${CUDA_TOOLKIT_ROOT_DIR}/lib64" "-lcudart_static" "-lculibos" "dl" "-pthread" "rt")
+  set(CMAKE_REQUIRED_INCLUDES ${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES})
+  #set(CMAKE_REQUIRED_LIBRARIES ${NVJPEG_LIBRARY} "-L${CUDA_TOOLKIT_ROOT_DIR}/lib64" "-lcudart_static" "-lculibos" "dl" "-pthread" "rt")
```

Comment:
Can be removed?

Reply:
In its current form, it definitely broke the build. If CI succeeds without it, I'll remove the line.